excelize.go

```diff
@@ -217,6 +217,60 @@ func OpenReader(r io.Reader, opts ...Options) (*File, error) {
 	return f, err
 }
 
+// StreamRead 流式读取，不支持加密的 Excel 2003
```

Comment:
Please don't introduce new exported function, just updated `OpenReader` function. Encrypted workbooks should be supported. Please also update unit tests for these changes.